lib/miq_smis_client.rb

```diff
@@ -682,14 +682,7 @@ def self.dotNodeInfo(node)
     end
 
     def self.dotHealth(node)
-      return 'springgreen3' # XXX
```

Comment:
@roliveri I don't know anything about the usage.  This line was added on "Mon Oct 31 19:40:10 2011", so I assumed that it was working as expected and we no longer needed the rest of the code.